catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -328,7 +328,8 @@ private void attemptAddRoutes() {
         "Attempting to add routes for content directory monitor watching {}", monitoredDirectory);
     try {
       RouteBuilder routeBuilder = createRouteBuilder();
-      verifyContentCamelComponentIsAvailable();
+      verifyContentCamelComponentIsAvailable("content");
+      verifyContentCamelComponentIsAvailable("catalog");
```

Comment:
✏️ Since this method can be used for several components now maybe call it "verifyCamelComponentIsAvailable()" instead of "verifyContentCamelComponentIsAvailable()".